tests/request_spec.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * What's the request in Kakapo? Check the example
+ *
+ * import {Server} from 'Kakapo';
+ * const server = new Server();
+ * server.get('/users', (request) => {
+ *   ...
+ * });
+ *
+ * It's an important part of Kakapo since contains useful information for 
+ * the user in the moment of returning the fake data from the handler.
+ * 
+ * So, even if the behaviour is implicity/explicity tested in other specs,
+ * this one must cover all the cases.
+ */
+import tapeTest from 'tape';
+import {Router} from '../src/kakapo';
+
+let router;
+
+function before() {
+  if (router && router.reset) {
+    router.reset();
+  }
+};
+
+function test(title, cb) {
+  tapeTest(title, (...args) => {
+    before();
+    cb(...args);
+  });
+}
+
+export default () => {
+  test('Request#headers', assert => {
+    assert.plan(2);
+    router = new Router();
+  
+    router.post('/users', (request) => {
+      assert.ok(request.headers.get('custom-request-header-1') === 'one', 'Receive the first header');
```

Comment:
Please change all assertion like this to something like:

```javascript
assert.ok(request.headers.get('custom-request-header-1') === 'one', 'Receive the first header');
```

Just for readability :)